content/docs/command-reference/add.md

```diff
@@ -344,3 +342,44 @@ $ tree .dvc/cache
 
 Only the hash values of the `dir/` directory (with `.dir` file extension) and
 `file2` have been cached.
+
+# Example: Transfer to remote storage
+
+When you want to include a remote location (like some file in a s3 bucket, or a
+Google Drive directory) to your project, but don't want DVC to control the given
+remote location rather just sync the data into your remote storage,
+`--to-remote` option can be used.
+
+```dvc
+$ mkdir example # workspace
+$ cd example
+$ git init
+$ dvc init
+$ mkdir /tmp/dvc-storage
+$ dvc remote add myremote /tmp/dvc-storage
+```
+
+Now let's add the `data.xml` to our remote storage, and create a `.dvc` file.
+
+```dvc
+$ dvc add https://data.dvc.org/get-started/data.xml --to-remote \
+          -r myremote
+```
+
+The only change in our local <abbr>workspace</abbr> is a newly created `.dvc`
```

Comment:
I would emphasize it a bit differently:

It behave the same way as a regular `dvc add` and create a `.dvc` file:


The only difference that dataset is transferred straight to remote, thus you don't need to ....